common/config/rush/pnpm-lock.yaml

```diff
@@ -5,742 +5,7 @@ importers:
   .:
     specifiers: {}
 
```

Comment:
It is very suspicious that there are that many lines removed and that little lines added to the lock file. Could you pull this repo to a fresh directory, revert the changes in pnpm-lock.yaml (download the file from master) and run `rush install`?